Editor/SnapZone/SnapZoneSettings.cs

```diff
@@ -49,6 +60,12 @@ public class SnapZoneSettings : ScriptableObject
         /// The material used for the highlight object. Should be transparent.
         /// </summary>
         public Material ValidationMaterial => SetupValidationMaterial();
+        
+        
+        /// <summary>
+        /// The material used for the highlight object. Should be transparent.
```

Comment:
Is this the real behavior? If so, what is the difference between `InvalidMaterial` and  `ValidationMaterial`?

Reply:
should probably be changed yes, InvalidMaterial will be shown when the validation says "not valid to be snapped"

Reply:
fixed